lib/plugins.js

```diff
@@ -190,6 +190,11 @@ function plugins (settings, deviceId) {
       .then(row => row.id)
   }
 
+  function fetchCurrentAvailableCoupons () {
+    const sql = `select * from coupons where soft_deleted=false`
```

Comment:
I've been marking lower case SQL in my recent reviews but I think we need to confirm if it's necesary with Rafael

Reply:
Yeah, better to standardize using upper case. "If you screen at the database it goes faster" :laughing: